src/main/java/net/dries007/tfc/compat/crafttweaker/CTHelper.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Work under Copyright. Licensed under the EUPL.
+ * See the project README.md and LICENSE.txt for more information.
+ */
+
+package net.dries007.tfc.compat.crafttweaker;
+
+import javax.annotation.Nonnull;
+
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fml.relauncher.ReflectionHelper;
+
+import crafttweaker.api.item.IItemStack;
+import crafttweaker.api.item.IngredientStack;
+import crafttweaker.api.liquid.ILiquidStack;
+import crafttweaker.api.oredict.IOreDictEntry;
+import net.dries007.tfc.objects.inventory.ingredient.IIngredient;
+
+public final class CTHelper
+{
+    @Nonnull
+    public static IIngredient getInternalIngredient(@Nonnull crafttweaker.api.item.IIngredient ingredient)
+    {
+        int amount = ingredient.getAmount();
+        if (ingredient instanceof IngredientStack)
+        {
+            //Yep, this is needed. This happens if user specifies ore dict with amount (ie: <ore:ingotSteel>*4).
+            //We can't map IngredientStack#getItems to ore dict otherwise
+            //That or we implement "IngredientMultipleItemStack"
```

Comment:
It't not unexpected. There are some inputs that could be ore dicted with a value, example: User wants to use any bronze ingot(from TFC or other mods) to craft bronze dust in quern, to do that using only one recipe registration he wants to use <ore:ingotBronze>*4. Because CT don't use `IOreDictEntry` with amounts, CT gives me `IngredientStack`, but block me access to the ore dict part. To actually recover the oredict string I need to use Reflection. when i said "IngredientMultipleItemStack", i'm refering to a new TFC IIngredient implementation that use a list of itemstacks instead of oredict. CraftTweaker's `IngredientStack` has a function called `getItemArray()` which gives me the "converted" ore dict -> itemstacks